packages/css/src/grid/grid.scss

```diff
@@ -6,10 +6,12 @@
 @import "../../utils/breakpoint";
 
 .amsterdam-grid {
+  --amsterdam-grid-gap: var(--amsterdam-grid-density-low-gap);
```

Comment:
Very cool that you're scoping variables but why would you when it doesn't change in the scope of this class? Since you do not nest the density high modifier? And also, is it not the _NLDS way_ to set this via tokens?